app/src/main/java/org/schabi/newpipe/local/bookmark/BookmarkFragment.java

```diff
@@ -66,11 +66,10 @@ public void onCreate(Bundle savedInstanceState) {
     public View onCreateView(@NonNull LayoutInflater inflater,
                              @Nullable ViewGroup container,
                              Bundle savedInstanceState) {
-        if (activity != null && activity.getSupportActionBar() != null) {
-            activity.getSupportActionBar().setDisplayShowTitleEnabled(true);
-            activity.setTitle(R.string.tab_subscriptions);
-        }
 
+        if(!useAsFrontPage) {
```

Comment:
Same as with mIsVisibleToUser. This should maybe be put into `BaseFrament` then onCreateView() should call its super.

Reply:
then onCreateView() should call its super.